components/hab/tests/test_install_script.ps1

```diff
@@ -0,0 +1,22 @@
+Describe "Install habitat using install.ps1" {
```

Comment:
There is not, because the `Install-Habitat` function performs this action https://github.com/habitat-sh/habitat/blob/master/components/hab/install.ps1#L140